certora/.gitignore

```diff
@@ -0,0 +1,5 @@
+# certora
```

Comment:
Is this file needed?

Reply:
Not really. We add these lines to the main gitignore. This is there for reference only. We can remove it if you prefer

Reply:
> * Whats the point of the `munged` directory? Is it not possible to use the contracts dependencies folder?

In some places in the contract we make changes like changing modifiers from internal to public, adding virtual, etc.
We use the munged folder in order to avoid polluting any of the original files, and keep all the changes contained within our own directory.
In general we try not to make too many changes by changing the original contract, in particular not any significant changes that may change the behavior of the methods.